go/chat/search/storage.go

```diff
@@ -130,9 +130,11 @@ func (s *store) getConvIndex(ctx context.Context, convID chat1.ConversationID, u
 			}
 		}
 		if err != nil {
+			s.Debug(ctx, "getConvIndex: failed to get index: %s", err)
 			if derr := s.deleteLocked(ctx, convID, uid); derr != nil {
 				s.Debug(ctx, "unable to delete: %v", derr)
 			}
+			err = nil
```

Comment:
we never really expect errors here i think it would be better to return them so we can a log if it happens in the wild